trellis/core/publisher.hpp

```diff
@@ -30,7 +30,15 @@ namespace core {
 template <typename MSG_T>
 class PublisherClass {
  public:
-  PublisherClass(const std::string& topic) : ecal_pub_(topic), ecal_pub_raw_(CreateRawPublisher(topic)) {}
+  PublisherClass(const std::string& topic) : PublisherClass(topic, false) {}
+
+  PublisherClass(const std::string& topic, bool enable_zero_copy)
```

Comment:
```suggestion
  PublisherClass(const std::string& topic, bool enable_zero_copy = false)
```

Reply:
Why do you prefer default argument vs. constructor overloading?

Reply:
Eh, not a big deal, it's just less code to maintain/update